### PR TITLE
Added styles for block quotes

### PR DESCRIPTION
**Pull request by patphongs**
Related to https://github.com/18F/fec-cms/pull/861

Implementation of issue #632

### CODE REVIEW

**Comment by jenniferthibault** [comment]
So exciting! I 😻  new style work.

A couple minor adjustments based on the [initial direction](https://cloud.githubusercontent.com/assets/11636908/22993813/6443ee5a-f392-11e6-9f3c-cced96332346.png) :
- Font style should be _italic_
- No need for the left padding between the grey bar and the container—the bar should off-set it more than enough that we shouldn't need to give it extra space. I did a quick sketch of what I mean based on your screenshot:
screen shot 2017-03-09 at 9 25 07 am

File: scss/elements/_typography.scss [hunk 1]
```diff
@@ -79,3 +79,9 @@ hr {
   border-top: none;
   margin: u(1rem 0);
 }
+
+blockquote {
+  border-left: 10px solid $gray-lightest;
+  padding-left: 30px;
+  font-size: u(1.6rem);
```
Comment (jenniferthibault) [inline scss/elements/_typography.scss]: This is actually something I just don't know, but maybe you can teach me :) 
If this is inheriting the `<p>` class styles, do we need to set the font size? Or should it have it automatically?

Comment (patphongs) [inline scss/elements/_typography.scss]: So the blockquote button is actually based inside of the paragraph wagtail component, however, the p tag does not actually wrap the blockquote so it does not inherit the paragraph styles. Which reminds me, I forgot to put in an image of what the blockquote button looks like in wagtail, see below.

screen shot 2017-03-09 at 9 50 21 am

**Comment by patphongs** [comment]
Thanks for your comments @jenniferthibault ! I've fixed it up according to your specs.

**Comment by jenniferthibault** [comment]
Thanks @patphongs ! 
Style review ✅ 
⛹️ passing to @xtine for technical review

**Pull request merged**